py4DSTEM/braggvectors/threshold.py

```diff
@@ -158,9 +158,9 @@ def universal_threshold(
         # Remove peaks that are too close together
         if maxNumPeaks is not False:
             r2 = minPeakSpacing**2
-            deletemask = np.zeros(pointlist.length, dtype=bool)
+            deletemask: npt.NDArray = np.zeros(pointlist.length, dtype=bool)
```

Comment:
Why are we adding a type hint here? I didn't even know you could do inline type hints like this!

Reply:
I didn't mean to include this, while valid code, I just wanted to see the state of type inferencing on linters but decided it wasn't worth it. I'll remove it and the import of npt